spidermon/contrib/actions/email/__init__.py

```diff
@@ -89,9 +89,7 @@ def run_action(self):
         if not self.fake:
```

Comment:
Maybe you could improve this if statement as well.
It is easier to read affirmative sentences and negative ones.

Reply:
Replace with affirmative condition.